x/data/client/tx.go

```diff
@@ -88,24 +90,31 @@ func MsgAttestCmd() *cobra.Command {
 				return err
 			}
 
-			iri := args[0]
-			if len(iri) == 0 {
-				return sdkerrors.ErrInvalidRequest.Wrap("iri is required")
-			}
-
 			attestor := clientCtx.GetFromAddress()
-			content, err := data.ParseIRI(iri)
-			if err != nil {
-				return sdkerrors.ErrInvalidRequest.Wrapf("invalid iri: %s", err.Error())
+
+			if len(args[0]) == 0 {
```

Comment:
I think this check is not needed - Corba is already doing it through line 86

Reply:
Removed in https://github.com/regen-network/regen-ledger/pull/971/commits/52315dd205bb31753d7d9cf127dd7c493deec89c

Reply:
Actually this prevents the error from being:

```
invalid iri: failed to parse IRI : regen: prefix required: invalid IRI: invalid request
```

And the error is instead:

```
iri is required
```